### PR TITLE
BaSP-TG-55: fix messages & status codes

### PR DESCRIPTION
**Pull request by julietapreliasco**
Small fixes in controllers folder regarding messages and delete status code

### CODE REVIEW

**Review by Romimonde — DISMISSED** [review]
Everything looks fine, thank you!

File: src/controllers/admins.js [hunk 1]
```diff
@@ -66,12 +66,12 @@ export const deleteAdmin = async (req, res) => {
...
     if (!admin) {
       throw new APIError({
-        message: 'Admin not found',
+        message: "Admin not found",
         status: 404,
       });
     }
-    return res.status(200).json({
-      message: 'Admin deleted',
+    return res.status(204).json({
```
Comment (lmalvasia) [inline src/controllers/admins.js:73]:
```suggestion
    return res.status(204);
```

File: src/controllers/employees.js [hunk 2]
```diff
@@ -68,11 +68,11 @@ export const deleteEmployees = async (req, res) => {
     const result = await Employees.findByIdAndDelete(req.params.id);
     if (!result) {
       throw new APIError({
-        message: 'Employee not found',
+        message: "Employee not found",
         status: 404,
       });
     }
-    return res.status(200).json({
+    return res.status(204).json({
```
Comment (lmalvasia) [inline src/controllers/employees.js:75]:
```suggestion
    return res.status(204);
```

File: src/controllers/super-admins.js [hunk 3]
```diff
@@ -67,17 +67,19 @@ export const createSuperAdmin = async (req, res) => {
...
       throw new APIError({
-        message: 'Super Admin not found',
+        message: "Super admin not found",
         status: 404,
       });
     }
 
-    return res.status(200).json({
-      message: 'Super admin was deleted',
+    return res.status(204).json({
```
Comment (lmalvasia) [inline src/controllers/super-admins.js:81]:
```suggestion
    return res.status(204);
```

File: src/controllers/tasks.js [hunk 4]
```diff
@@ -65,12 +65,12 @@ export const deleteTask = async (req, res) => {
...
     if (!task) {
       throw new APIError({
-        message: 'Task not found',
+        message: "Task not found",
         status: 404,
       });
     }
-    return res.status(200).json({
-      message: `Task with id ${req.params.id} deleted`,
+    return res.status(204).json({
```
Comment (lmalvasia) [inline src/controllers/tasks.js:72]:
```suggestion
    return res.status(204);
```

**Review by lmalvasia — APPROVED** [review]

**Pull request merged**